### PR TITLE
Eliminate couple default conversions

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/82484

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func optionsForTransform(mediaType negotiation.MediaTypeOptions, req *http.Reque
 	case target == nil:
 	case target.Kind == "Table" && (target.GroupVersion() == metav1beta1.SchemeGroupVersion || target.GroupVersion() == metav1.SchemeGroupVersion):
 		opts := &metav1beta1.TableOptions{}
-		if err := metav1beta1.ParameterCodec.DecodeParameters(req.URL.Query(), metav1beta1.SchemeGroupVersion, opts); err != nil {
+		if err := metainternalversionscheme.ParameterCodec.DecodeParameters(req.URL.Query(), metav1beta1.SchemeGroupVersion, opts); err != nil {
```
Comment (wojtek-t) [inline staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go]: @smarterclayton - can you please confirm that this is correct (all tests are passing, but I want to ensure that I didn't miss any nuance with it)

Comment (smarterclayton) [reply staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go]: Hrm.  I think it's correct.  I did this deliberately, and the set of tests should be sufficient.

**Pull request merged**